controllers/booksController.js

```diff
@@ -16,9 +16,28 @@ export const handleNewBook = async (req, res) => {
     const result = await Book.create({
       'title': title,
       'author': author,
+      'readOn': readOn
     });
     res.status(201).json({ 'success': `New book ${title} added!` });
   } catch (err) {
     res.status(500).json({ 'message': err.message });
   }
 };
+
+export const handleGetBooks = async (req, res) => {
```

Comment:
I've been following this naming convention of using `handle` as the prefix for all of these functions, I don't love it but I think that might be typical for node?